tests/_fixtures/cosmos/package-describe.json

```diff
@@ -1,18 +1,19 @@
 {
```

Comment:
This file was one year old therefore outdated. I took the updated response for the same package.